### ISSUE TITLE
Remove nonfunctional prereleases

### ISSUE THREAD

**Issue by LexiconCode** [dictation-toolbox/natlink#103]
It would be best if we removed or hid any releases that aren't functional. They may be a pre-release but it still causes confusion when they don't work as expected we want to minimize that. You can only ask so much of the community to be a beta or alpha tester.

**Comment by quintijn** [comment]
Well... if people take the latest release...

Now in the releases list, also the changes are maintained. Are they kept elsewhere too?? Otherwise, when removing previous (pre) releases, we loose the list of changes.

**Comment by LexiconCode** [comment]
The changes are the changes are in the commit and pull request history. If everything is done by a pull request its auto change log and generated for you. See [natlinkv5.0.0.pre](https://github.com/dictation-toolbox/natlink/releases/tag/natlink5.0.0.pre) as an example.

When the time comes we can do another pre-release starting at https://github.com/dictation-toolbox/natlink/commit/9de7b63eb689d2af5a724e6d429266421574197a and it'll grab all the changes until the latest. Then we can combine all the prereleases into one release and have it reflect all the changes. 

And it's just a matter of combining the summary from each logs making sure it's nice and succinct.

The bottom line is we need to really make sure were not releasing even in a prerelease something that is nonfunctional. If that's the case then we should pull it and redo a release.
